### DISCUSSION TITLE
How to generate the camera path file without GUI?

### DISCUSSION THREAD

**Discussion by X-Shi** [NVlabs/instant-ngp#540] [Q&A]
I want to generate videos with a sequence of images with different camera positions. However, all I find from the closed issues need to load a camera path .json file first. I am running the codes without GUI. Building an interactive GUI is not possible for me. I am wondering how to generate that camera path file without GUI?

**Comment by benben516252275** [comment]
I also met this problem，any solution？

**Reply by 3a1b2c3** [reply]
I hand edited myself a json file to use with the render script before buying a gtx, because of the strange coordinate system would not recommend 👯

**Comment by hieuristic** [comment]
[#678](https://github.com/NVlabs/instant-ngp/issues/678) seems to generate cam.json from transforms.json
